### PR TITLE
Forgot Code

### PR DESCRIPTION
**Pull request by HahaYes-dev**
Dear squid, I'm not sure about whether this code is right, but I used std::string in it. Please tell me if it is wrong and I'll correct it in the future.

### CODE REVIEW

File: src/Lexer.h [hunk 1]
```diff
@@ -71,6 +71,18 @@ void AddKeyword(night::array<Token>& tokens, night::string& token, TokenType&& t
 	token = "";
 }
 
+void easter_egg(std::string& token){
```
Comment (DynamicSquid) [inline src/Lexer.h:74]: use `night::string` instead;

File: src/Lexer.h [hunk 2]
```diff
@@ -71,6 +71,18 @@ void AddKeyword(night::array<Token>& tokens, night::string& token, TokenType&& t
...
+	if(token = "squid")
```
Comment (DynamicSquid) [inline src/Lexer.h:75]: use `night::string` instead, and you have to use double equal signs: `==` for comp

**Comment by DynamicSquid** [comment]
Eh, I don't really feel like adding an easter egg now...

**Comment by firefish111** [comment]
Stop it, We don't like ascii art, not of a spray-paint can, not of a language @HahaYes-dev

**Pull request closed without merging**